examples/maxnum/player.go

```diff
@@ -40,7 +40,7 @@ func (player *Player) Reduce(message phi.Message) phi.Message {
 			}
 			if uint(len(player.seen)) == player.players {
 				done := Done{player: player.id, max: player.currentMax}
-				return phi.MessageBatch{Messages: []phi.Message{message, done}}
+				return phi.Messages{Messages: []phi.Message{message, done}}
```

Comment:
If we have defined `phi.Messages` as `type Messages []Message` then this can become `return phi.Messages{ message, done }` which is much more concise.

Reply:
Fixed in 492f01838805626407d5dbab9206c3b1d9c936ab